mediafire.py

```diff
@@ -469,12 +469,12 @@ def download_file(
 
             response = conn.getresponse()
 
-    if response.status != 200:
+    '''if response.status != 200:
```

Comment:
replace `if response.status != 200` with `if 400 <= response.status < 600` and check if it works with your link instead of commenting everything out